Model/Indexer/Product.php

```diff
@@ -62,6 +62,10 @@ public function execute($productIds)
             return;
         }
 
+        if ($productIds) {
+            $productIds = array_merge($productIds, $this->productHelper->getParentProductids($productIds));
```

Comment:
Typo, method name is `getParentProductIds` :)

Reply:
Sharp eye :)